.goreleaser.template.yml

```diff
@@ -0,0 +1,50 @@
+env:
+- GOPROXY=https://proxy.golang.org|direct
+- GO111MODULE=on
+- CGO_ENABLED=0
+before:
+  hooks:
+    - go mod tidy
+    - go mod download
+builds:
+  - id: operator-controller
+    main: ./
+    binary: bin/manager
+    tags: $GO_BUILD_TAGS
+    goos:
+    - linux
```

Comment:
Not sure if GH ate my comment or if I managed not to submit it.

This seems to build only a single binary for `linux/$GOARCH`. But we need to build binaries for the following:
- `linux/amd64`
- `linux/arm64`
- `linux/ppc64le`
- `linux/s390x`

Reply:
We then need to propagate these to separate individual docker images, and then build the manifest list that combines all of them.

Reply:
All this is probably fine to do in a follow-up, but might be good to do that before we tag this repo the first time.